CMakeLists.txt

```diff
@@ -351,13 +351,41 @@ set(WABT_LIBRARY_SRC
 )
 
 add_library(wabt STATIC ${WABT_LIBRARY_SRC})
+add_library(wabt::wabt ALIAS wabt)
+
+target_compile_features(wabt PUBLIC cxx_std_17)
```

Comment:
How is this different to the existing `set(CMAKE_CXX_STANDARD 17)`?

Reply:
It's a requirement that propagates to consumers... CMake will pick the highest standard version among the transitively linked set of targets and issue an error if the active toolchain is incapable of compiling it.